tooltester/src/main/resources/PipelineTest.groovy

```diff
@@ -1,4 +1,4 @@
-currentBuild.displayName = "1.4.0-beta.3"
+currentBuild.displayName = "1.5.0-alpha.0"
```

Comment:
Seems out of date with tooltester/pom.xml?
Can it use the same property?

Reply:
So this file isn't actually used to run Jenkins. it's sort of a backup in case Jenkins explodes.  I should be able to just make the displayName a build parameter that is sent to Jenkins instead of hardcoding it here/there.  I'll make a personal TODO for this.